### PR TITLE
[WIP] Reenable GKE test in Node tests

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/kubernetes/issues/98326

This is supposed to be no-op.

```release-note
NONE
```

/kind cleanup
/priority important-soon

### CODE REVIEW

File: test/e2e_node/e2e_node.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
+// +build linux
+
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (lala123912) [inline test/e2e_node/e2e_node.go:4]: 2016 ->2020

**Comment by SergeyKanzhelev** [comment]
@wojtek-t do you plan to proceed with this? Referenced bug is closed now so I'm closing PR. Please reopen when you want to work on it.

**Pull request closed without merging**